src/nuklear_font.c

```diff
@@ -190,7 +190,9 @@ nk_font_bake_pack(struct nk_font_baker *baker,
     /* setup font baker from temporary memory */
     for (config_iter = config_list; config_iter; config_iter = config_iter->next) {
         it = config_iter;
-        do {if (!stbtt_InitFont(&baker->build[i++].info, (const unsigned char*)it->ttf_blob, 0))
+        struct stbtt_fontinfo *font_info = &baker->build[i++].info;
```

Comment:
`stb_truetype` passes the `userdata` field of `stbtt_fontinfo` as the second argument to `STBTT_malloc`/`STBTT_free`.
Set the `userdata` here to point to the allocator (`alloc`), and use it in the custom implementations of `STBTT_malloc`/`STBTT_free`.